shell/platform/embedder/embedder.h

```diff
@@ -1321,6 +1330,11 @@ FlutterEngineResult FlutterEngineSendWindowMetricsEvent(
     FLUTTER_API_SYMBOL(FlutterEngine) engine,
     const FlutterWindowMetricsEvent* event);
 
+FLUTTER_EXPORT
+FlutterEngineResult FlutterEngineSendScreenMetricsEvent(
+    FLUTTER_API_SYMBOL(FlutterEngine) engine,
+    const FlutterScreenMetricsEvent* event);
```

Comment:
Since we don't make breaking changes to embedder.h, we should be as future-proof as possible in any new API we're adding. That means we shouldn't add an API that assumes a single screen when we know we're going to need to allow a list of screens. Instead we should design it for the future case, and document that it currently only uses the first screen.